py-polars/docs/source/reference/dataframe/index.rst

```diff
@@ -19,6 +19,8 @@ This page gives an overview of all public DataFrame methods.
    plot
    style
 
+.. _dataframe:
```

Comment:
What does this do?

Reply:
It's a Sphinx internal reference anchor; you'll see them in a variety of places elsewhere in the docs 👌